code/modules/integrated_electronics/core/detailer.dm

```diff
@@ -25,7 +25,9 @@
 		"green" = COLOR_ASSEMBLY_GREEN,
 		"light blue" = COLOR_ASSEMBLY_LBLUE,
 		"blue" = COLOR_ASSEMBLY_BLUE,
-		"purple" = COLOR_ASSEMBLY_PURPLE
+		"purple" = COLOR_ASSEMBLY_PURPLE,
+		"pink" = COLOR_ASSEMBLY_PINK,
+		"custom" = "#ffffff"
```

Comment:
```suggestion
		"custom" = COLOR_ASSEMBLY_WHITE
```

Reply:
Fair enough, looks prettier, that color is never used though